libkbfs/folder_update_prepper.go

```diff
@@ -1095,7 +1095,8 @@ func (fup folderUpdatePrepper) prepUpdateForPaths(ctx context.Context,
 		// Ignore it if it doesn't descend from an original block
 		// pointer or one created in the merged branch.
 		if _, ok := unmergedChains.originals[update.Unref]; !ok &&
-			unmergedChains.byOriginal[update.Unref] == nil &&
+			(unmergedChains.byOriginal[update.Unref] == nil ||
+				unmergedChains.isCreated(update.Unref)) &&
```

Comment:
When a block is created in a branch, but has an "update" in the resolution op, we always want to ignore it.  Normally in CR there wouldn't be an update, but for `SyncAll()` things are a bit different because we don't squash create and sync operations together like we do in CR, so there will be a syncOp update for a newly-created file.